src/shaders/shaders.js

```diff
@@ -86,6 +86,13 @@ export const symbolTextAndIcon = compile(symbolTextAndIconFrag, symbolTextAndIco
 
 function compile(fragmentSource, vertexSource) {
     const re = /#pragma mapbox: ([\w]+) ([\w]+) ([\w]+) ([\w]+)/g;
+    const attributeRegex = /attribute ([\w]+) ([\w]+)/g;
+    const uniformRegex = /uniform ([\w]+) ([\w]+)([\s]*)([\w]*)/g;
```

Comment:
hmm.. Uniform regex is being use multiple times.. It feels more clean to me to have them separately since they are very different in terms of Regex which makes it easier to later modify it if we want to.. 
But I don't have a strong preference.